### ISSUE TITLE
The program window starts and hangs, it is impossible to close the program (windows 10)

### ISSUE THREAD

**Issue by DatiQ** [adobe/brackets#13537]
The program window starts and hangs, it is impossible to close the program
(windows 10)

**Comment by rocket-martue** [comment]
I have same issue.

**Comment by MaizeStarch** [comment]
me 2, this is a mess

**Comment by MaizeStarch** [comment]
I'm in Windows 7

**Comment by beito123** [comment]
me too... (win10)

**Comment by 247handa** [comment]
I had a same problem on OS X 10.11.6 El Capitan, but I finally fixed it after deleting brackets-shizimily-multiencoding.
Don't you use any extensions related multi encoding?

**Comment by assialiholic** [comment]
I had a same problem on OS X 10.11.6 El Capitan, but I finally fixed it after deleting brackets-shizimily-multiencoding.
Don't you use any extensions related multi encoding?

**Comment by saurabh95** [comment]
Yeah, we have now added multiple encoding support in the core brackets itself.

**Comment by beito123** [comment]
Thanks, I fixed it after removed the extension.

**Comment by DatiQ** [comment]
THX!!

**Comment by AtaNurchin** [comment]
my system is win 10 but brackets 1.10 program isnt working (any functions working for me or all 👍 )
and where is the brackets-shizimily-multiencoding extention folder ?

**Comment by saurabh95** [comment]
The brackets-shizimily-multiencoding folder is located in `%appdata%\Brackets\extensions\user`

**Comment by assialiholic** [comment]
Just in case, here are win's and mac's full path.
・Windows
C:\Users\USERNAME\AppData\Roaming\Brackets\extensions\user
・Mac
/Users/USERNAME/Library/Application Support/Brackets/extensions/user

Good luck :)

**Comment by rocket-martue** [comment]
@saurabh95  Thanks!!

**Comment by AhmedBHameed** [comment]
Version 1.10 unable to start correctly (0xc000007b)!!!!
going down to version 1.9...

**Comment by anephew** [comment]
I'am using previously brackets-shizimily-multiencoding. After upgrade to 1.10 i got the same error. Then i deleted multiencoding and run successfully Brackets, but now after opening any file and edit them - they are not saved (neither via keyboard shortcut, or via menu)!

**Comment by loorlab** [comment]
Hi Friends, After an open time the program closes automatically.

Version 1.10  - Win10

Help Us !

**Comment by loorlab** [comment]
Thanks @saurabh95 Although my problem is different because I if we can open the program but then it collapses, however we reinstalled it.

Let's look at whether this is still happening this week.

**Comment by PanianPazan** [comment]
Thanks @assialiholic!!!!
It's working again!!
